src/main/java/com/iridium/iridiumskyblock/IridiumSkyblock.java

```diff
@@ -513,6 +516,34 @@ public void loadConfigs() {
         Bukkit.getPluginManager().callEvent(reloadEvent);
     }
 
+    private void updateBackButtons() {
+        BackButton backButton22Is = new BackButton(true, "is", new Item(XMaterial.NETHER_STAR, 22, 1, "&c&lBack", Collections.emptyList())); //Used for making inline if statements
```

Comment:
just have a BackButton that is relative to the inventory size, e.g. have a slot of -5 which will put it in the slot inventorysize-5